imap_processing/swe/tests/test_swe_housekeeping.py

```diff
@@ -0,0 +1,107 @@
+import pandas as pd
+import pytest
+
+from imap_processing import imap_module_directory
+from imap_processing.swe.l0 import decom_swe
+
+
+@pytest.fixture(scope="session")
+def decom_test_data():
+    """Read test data from file"""
+    packet_1_file = (
+        f"{imap_module_directory}/swe/tests/20230927100248_SWE_HK_packet.bin"
+    )
+    packet_2_file = (
+        f"{imap_module_directory}/swe/tests/20230927100412_SWE_HK_packet.bin"
+    )
+    first_data = decom_swe.decom_packets(packet_1_file)
+    second_data = decom_swe.decom_packets(packet_2_file)
+
+    return first_data + second_data
+
+
+def test_swe_raw_housekeeping_data(decom_test_data):
+    """This test and validate raw and derived data of SWE housekeeping data."""
+    # read validation data
+    test_data_path = f"{imap_module_directory}/swe/tests"
+    raw_validation_data = pd.read_csv(
+        f"{test_data_path}/data_raw.SWE_APP_HK_20230927_094839.csv",
+        index_col="SHCOARSE",
+    )
+
+    data = decom_test_data[0]
+    validation_data = raw_validation_data.loc[data.data["SHCOARSE"].raw_value]
+
+    # should have two packets
+    assert len(decom_test_data) == 2
+
+    # compare raw values of housekeeping data
+    for key, value in data.data.items():
+        if key == "SHCOARSE":
+            # compare SHCOARSE value
+            assert value.raw_value == validation_data.name
+            continue
+        # check if the data is the same
+        assert value.raw_value == validation_data[key]
+
+
+def test_swe_derived_housekeeping_data(decom_test_data):
+    """This test and validate derived data of SWE housekeeping data."""
+    # read validation data
+    test_data_path = f"{imap_module_directory}/swe/tests"
+    derived_validation_data = pd.read_csv(
+        f"{test_data_path}/data_derived.SWE_APP_HK_20230927_094839.csv",
+        index_col="SHCOARSE",
+    )
+    data = decom_test_data[1]
+    validation_data = derived_validation_data.loc[data.data["SHCOARSE"].raw_value]
+
+    # Check ENUM values from housekeeping data
```

Comment:
In the other tests you are looping and asserting, do you want to do that here too?
```python
for key in ["APP_MODE", "SAFED", ...]:  # or data.data.keys()?
    assert data.data[key].derived_value == validation_data[key]
```

Reply:
I can do list with selected keys because I wanted to check enums value only in this case.